### PR TITLE
docs(appmesh): comprehensive App Mesh readme review

### PR DESCRIPTION
**Pull request by Seiya6329**
Conducting comprehensive review on `README` file.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by dfezzie** [comment]
Looks good to me. I don't see anything obviously wrong. Good to get it updated with all the final API changes

**Comment by Seiya6329** [comment]
@dfezzie Thanks for reviewing the PR!

File: packages/@aws-cdk/aws-appmesh/README.md [hunk 1]
```diff
@@ -35,17 +35,17 @@ After you create your service mesh, you can create virtual services, virtual nod
 The following example creates the `AppMesh` service mesh with the default egress filter of `DROP_ALL`. See [the AWS CloudFormation `EgressFilter` resource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-mesh-egressfilter.html) for more info on egress filters.
 
 ```ts
-const mesh = new Mesh(stack, 'AppMesh', {
-  meshName: 'myAwsmMesh',
+const mesh = new appmesh.Mesh(stack, 'AppMesh', {
```
Comment (madeline-k) [inline packages/@aws-cdk/aws-appmesh/README.md:38]: Remove the `appmesh` qualifier here and everywhere in the README. Types from the documented module should be unqualified. To make that compile, add a default fixture in this module that imports all the appmesh types that are needed. 

Details on Rosetta fixtures: https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#rosetta 

An example module-level default fixture: https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-kinesisfirehose/rosetta/default.ts-fixture

**Review by madeline-k — DISMISSED** [review]
Looks great! Just one comment about using module qualifiers in code snippets.

File: packages/@aws-cdk/aws-appmesh/README.md (hunk 1 above)
Comment (Seiya6329) [reply packages/@aws-cdk/aws-appmesh/README.md]: Thanks for attaching a link to the relevant to document!

**Review by madeline-k — APPROVED** [review]
The README looks good! 

The PR build looks to be failing due to some transient failure in another module. A re-run will hopefully resolve this. After that, you might need to add a rosetta fixture for the build to succeed. To test code snippet compilation locally, you can run: `yarn rosetta:extract`.

**Comment by Seiya6329** [comment]
Thanks @madeline-k !

Looks like there is no auto-merge and auto-rebuild happening.
I will check the status and revert back to you to get another round of approval.

**Review by madeline-k — APPROVED** [review]

**Pull request merged**